### ISSUE TITLE
cef3 for Java 64

### ISSUE THREAD

**Issue by tbhasme** [wjywbs/javacef#38]
We have an application that runs on 64 bit java. Since cef3 runs on 32 bit java, it isn't compatible with our application. Is there any way to use cef3 for 64 bit java? I assume all the dlls being used will be changed.

**Comment by PhilAndrew** [comment]
It says on the home page of this git "javacef 3 Stable on Windows, Mac and Linux with 32-bit and 64-bit support."

**Comment by tbhasme** [comment]
Yep, win64, but not java 64. All the libraries being used are for 32 bit java which fail to load in a 64 bit JRE.

[PTC Logo]http://www.ptc.com/

Tushar Bhasme
Technical Lead

909.604.6349
tbhasme@ptc.com

ptc.comhttp://www.ptc.com
[Twitter Icon]http://twitter.com/PTC   [Facebook Icon] http://www.facebook.com/PTC.Inc    [Linked In Icon] http://www.linkedin.com/company/ptc    [YouTube Icon] http://www.youtube.com/user/ptcstudio    [Instagram Icon] https://www.instagram.com/ptc_inc/

**Comment by PhilAndrew** [comment]
Well, this one works in SWT, the other one here https://bitbucket.org/chromiumembedded/java-cef works in AWT. If your willing to go to AWT then that's a work-around, in any case this is a bug.

**Comment by tbhasme** [comment]
Yep, tried that too. But there are some bugs (already raised in the forum) that are a concern.

**Comment by wjywbs** [comment]
There are 64 bit java libraries in this project to run under 64 bit java. You'll need to select to build 64 bit in configure.bat (CEF3Instructions.md).

https://github.com/wjywbs/javacef/blob/master/cef3/java/lib/win/swt64.jar

**Comment by schandur** [comment]
Hi there @wjywbs , could you please let me know which branch is the "stable" version of javacef3? I tried 3.1750 but the Sample browser SWT app does not run.

With 3.2526, using ninja, I ran into compile errors (missing .cc files)

**Comment by wjywbs** [comment]
The master branch is stable, and 3.2526 is experimental. What errors did you see when you ran 3.1750?

**Comment by schandur** [comment]
An access violation in AppInitCommandLine, in the C++ sources.

**Comment by schandur** [comment]
To give additional context, I'm running Java 8, 64-bit on Windows. This is the exception:

`
A fatal error has been detected by the Java Runtime Environment:

 EXCEPTION_ACCESS_VIOLATION (0xc0000005) at pc=0x000007fef27b2962, pid=22904, tid=17440

JRE version: Java(TM) SE Runtime Environment (8.0_74-b02) (build 1.8.0_74-b02)
Java VM: Java HotSpot(TM) 64-Bit Server VM (25.74-b02 mixed mode windows-amd64 compressed oops)
Problematic frame:
C  [chromium_loader.dll+0x2962]  AppInitCommandLine+0x92

Failed to write core dump. Minidumps are not enabled by default on client versions of Windows
`
